README.md

```diff
@@ -26,7 +38,213 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
-# <a name="getting-started"></a> Getting Started
+## <a name="azure"></a>Azure
+
+## <a name="azure-getting-started"></a> Getting Started
+
+You can find more information about every step below in other categories. This list of commands show how to get started quickly. Please see other items below to understand details and solve any problems you face.
+
+You should have `az cli` in your local to continue.
+
+You should use `ssh-agent` to add your ssh keys, which will be used for downloading the enterprise repository. Note that your keys are kept only in memory, therefore this is a secure step.
+
+In `azuredeploy.parameters.json` file, you will see the parameters that you can change. For example if you want to change the number of instances, you will need to change the parameter `numberOfInstances`. You can change the type of coordinator and workers separately from the parameters file. Also by default for workers, memory intense vms are used(E type) while for coordinator CPU intense vms are used(D type).
+
+After you run tests, you can see the results in `results` folder. The `results` folder will have the name of the config used for the test.
+
+## <a name="azure-setup-steps"></a> Setup Steps For Each Test
+
+You will need to follow these steps to create a cluster and connect to it, on your local machine:
+
+```bash
+
+# start ssh agent
+eval `ssh-agent -s`
+
+# Add your Github ssh key for enterprise (private) repo
+ssh-add
+
+# Put your public ip to sshPublicKey in azuredeploy.parameters.json. You can see your public key with:
+cat ~/.ssh/id_rsa.pub
+
+# in the session that you will use to ssh, set the resource group name
+export RESOURCE_GROUP_NAME=talha_test_resource_group
+
+# Quickly start a cluster of with defaults. This will create a resource group and use it for the cluster.
+./create-cluster.sh
+
+# Delete security rule 103 to be able to connect
+./delete_security_rule.sh
+
+# When your cluster is ready, it will prompt you with the connection string, connect to coordinator node
+ssh -A pguser@<public ip of coordinator>
+```
+
+## <a name="azure-delete-cluster"></a> Steps to delete a cluster
+
+After you are done with testing you can run the following the delete a cluster and the relevant resource group:
+
+```bash
+# Delete the formation
+# It's a good practice to check deletion status from the azure console
+./delete_resource_group.sh
+```
+
+## <a name="azure-basic-cluster-setup"></a> Basic Cluster Setup
+
+On the coordinator node:
+
+```bash
+# Setup your test cluster with PostgreSQL 11.5 and Citus master branch
+fab use.postgres:11.5 use.citus:master setup.basic_testing
```

Comment:
I guess these instructions are not specific to Azure? Should there be a common section on how to use fab?

Reply:
No they are not. At the end of the readme there is a section that explains the usage of fab. This section existed in AWS so I have copied it for Azure, but this is not necessary. We can remove it if it creates a confusion.

Reply:
btw, this seems to install postgres multiple times.

Reply:
Having a common section would be good.

Reply:
Agreed, this is too much repetition now as I already noted in my first review as well

Reply:
This is fixed